CMakeLists.txt

```diff
@@ -20,7 +20,12 @@ INCLUDE_DIRECTORIES(${JAVA_INCLUDE_PATH2})
 
 add_library(perfmap SHARED src/c/perf-map-agent.c src/c/perf-map-file.c)
 
-find_package(Java REQUIRED)
+find_package(Java COMPONENTS Development)
+if (Java_Development_FOUND)
+    message("Java development tools found.")
+else()
+    message(FATAL_ERROR "Java development tools not found.")
+endif()
 include(UseJava)
 
 set(CMAKE_JAVA_INCLUDE_PATH ${JAVA_INCLUDE_PATH}/../lib/tools.jar)
```

Comment:
Is this still needed then?

Reply:
yes, if running Java 8 or earlier. Does not hurt with newer, just references a non-existing file.